src/mscorlib/model.xml

```diff
@@ -7297,6 +7297,7 @@
       <Member Name="EndsWith(System.String)" />
       <Member Name="EndsWith(System.String,System.Boolean,System.Globalization.CultureInfo)" />
       <Member Name="EndsWith(System.String,System.StringComparison)" />
+      <Member Name="EndsWith(System.Char)" />
```

Comment:
Nit: Do we try to keep these sorted? Should the `System.Char` overload be listed above the `System.String` overloads?